### PR TITLE
#419 Add Change LineStyleType in Batch

### PR DESCRIPTION
**Pull request by langonginc**

### CODE REVIEW

**Review by thekingofcity — CHANGES_REQUESTED** [review]
Some text and translations should be updated. Also, type casting is not need :)

File: src/i18n/translations/ja.json [hunk 1]
```diff
@@ -326,6 +326,12 @@
                     "changeTo": "このタイプに変更する：",
                     "info": "駅のタイプを変更すると、位置と名前以外の特定の属性がすべて削除されます。変更する前に保存してください！"
                 },
+                "changeLineStyleType": {
+                    "title": "일괄적으로 라인 유형 변경",
```
[the remark this answers is not in the document]
Comment (langonginc) [reply src/i18n/translations/ja.json:330]: It's an accident 🌚

**Comment by langonginc** [comment]
Thanks, I'll fix the translation.

**Review by thekingofcity — APPROVED** [review]
LGTM

**Pull request merged**